src/database/mod.rs

```diff
@@ -325,6 +327,69 @@ impl Database {
         let result = self.pool.fetch_one(query).await?;
         Ok(result.get::<i64, _>(0) as i32)
     }
+
+    pub async fn get_provers(&self) -> Result<prover::Provers, Error> {
+        let query = sqlx::query(
+            r#"
+                SELECT batch_size, url, timeout_s
+                FROM provers
+            "#,
+        );
+
+        let result = self.pool.fetch_all(query).await?;
+
+        Ok(result
+            .iter()
+            .map(|row| {
+                let batch_size = row.get::<i64, _>(0) as usize;
+                let url = row.get::<String, _>(1);
+                let timeout_s = row.get::<i64, _>(2) as u64;
+                prover::Prover {
+                    url,
+                    batch_size,
+                    timeout_s,
+                }
+            })
+            .collect::<prover::Provers>())
+    }
+
+    pub async fn write_prover(
```

Comment:
nit: `insert_prover` or even `insert_prover_configuration` to be more consistent with other db methods